pkg/nodes/node.go

```diff
@@ -189,6 +188,46 @@ func (builder *NodeBuilder) ExternalIPv4Network() (string, error) {
 	return extNetwork.IPv4, nil
 }
 
+// IsReady check if the Node is Ready.
+func (builder *NodeBuilder) IsReady() bool {
+	if valid, _ := builder.validate(); !valid {
+		return false
+	}
+
+	glog.V(100).Infof("Verify %s node availability", builder.Definition.Name)
+
+	if !builder.Exists() {
+		return false
+	}
+
+	for _, condition := range builder.Object.Status.Conditions {
+		if condition.Type == "Ready" {
+			return condition.Status == isTrue
+		}
+	}
+
+	return false
+}
+
+// Get fetches the defined Node from the cluster.
+func (builder *NodeBuilder) Get() (*v1.Node, error) {
```

Comment:
I think you can safely remove this function from commit. PullNode or Exists safely pull latest node struct in to builder.Object that you can later use in test cases.